indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -0,0 +1,208 @@
+            {% endfor %}
+            {% if can_comment or can_review %}
+                {{ render_comment_box(abstract, form) }}
+            {% endif %}
+        </div>
+    {% endif %}
+{% endmacro %}
+
+
+{% macro _render_comment(comment) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action"></div>
+        <div class="i-timeline-item-box header-indicator">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=comment.user.name -%}
+                        <strong>{{ name }}</strong> left a comment
+                    {%- endtrans %}
+                    {{ render_visibility_hint(comment.visibility) }}
+                    <time datetime="{{ comment.created_dt.isoformat() }}">
+                        {{- comment.created_dt|format_human_date -}}
+                    </time>
+                    {% if comment.modified_dt %}
+                        {{ render_edited_hint(comment.modified_dt, comment.modified_by) }}
+                    {% endif %}
+                </div>
+                {% if comment.can_edit(session.user) %}
+                    <div class="abstract-comment-badges">
+                        <a class="i-button-icon icon-edit js-edit-comment"
+                           title="{% trans %}Edit comment{% endtrans %}"
+                           data-href="{{ url_for('.edit_abstract_comment', comment) }}"
+                           data-form-container="#comment-{{ comment.id }}"
+                           data-update=".management-page"
+                           data-replace-update
+                           data-confirm-close-unsaved
+                           data-ajax-form></a>
+                        <a class="i-button-icon icon-cross js-delete-comment"
+                           title="{% trans %}Remove comment{% endtrans %}"
+                           data-method="DELETE"
+                           data-href="{{ url_for('.delete_abstract_comment', comment) }}"
+                           data-title="{% trans %}Remove comment{% endtrans %}"
+                           data-confirm="{% trans %}Are you sure you want to remove this comment?{% endtrans %}"
+                           data-ajax>
+                        </a>
+                    </div>
+                {% endif %}
+            </div>
+            <div id="comment-{{ comment.id }}" class="i-box-content">
+                {{ comment.text }}
+            </div>
+        </div>
+    </div>
+{% endmacro %}
+
+
+{% macro _render_review(review, form, management=false) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action abstract-comment-pic"></div>
+        <div class="i-timeline-item-box header-indicator review-results">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=review.user.name -%}
+                        <strong>{{ name }}</strong> left a review
```

Comment:
"left a review" sounds weird to me. Like the reviewer dumping something on the abstract :hankey: :D